Makefile

```diff
@@ -54,14 +54,21 @@ test-unit:
 
 test-integration:
 	# Runs all tests including integration and property tests
+	# We don't currently have any integration tests, but we're leaving this so we can add more.
 	go test -race -tags="integration property_test" $$(go list ./... | grep -v /vendor/ | grep -v /cmd/)
 
 testlocal: generate test
 
+swarmtest:
+	go install github.com/scootdev/scoot/binaries/setup-cloud-scoot
+	${GOPATH}/bin/setup-cloud-scoot --strategy local.local run ${GOPATH}/bin/scootapi run_smoke_test 10 1m
```

Comment:
We have to be careful with GOPATH here and elsewhere. I have multiple paths in mine. Why do we need the go install bit anyway?